3-Write-your-own-network_en.ipynb

```diff
@@ -39,6 +39,8 @@
     "import chainer.functions as F\n",
     "import chainer.links as L\n",
     "\n",
+     "gpu_id = 0  # Set to -1 if you don't have a GPU\n",
```

Comment:
I think it isn't needed to be placed here. I think it's better to put this line right before the line which performs model instantiation (`model = train(MyModel(10), gpu_id=gpu_id))`). How do you think about this?